src/ImageProcessor.Web.Plugins.PostProcessor/PostProcessor.cs

```diff
@@ -99,16 +98,17 @@ public static async Task<MemoryStream> PostProcessImageAsync(HttpContext context
                         }
                     }
 
-                    // Save result
+                    // Use source file if no destination file is created
                     if (!File.Exists(destinationFile))
                     {
-                        File.Copy(sourceFile, destinationFile, true);
+                        destinationFile = sourceFile;
```

Comment:
No need to copy the file, as we can just use the source file path as destination 😄